providers/statsig/pom.xml

```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>com.statsig</groupId>
             <artifactId>serversdk</artifactId>
-            <version>1.22.0</version>
+            <version>1.18.1</version>
```

Comment:
This seems to have a breaking change, and due to the actions issues in github last week, [got through](https://github.com/open-feature/java-sdk-contrib/pull/822). Reverting.